app/services/users.js

```diff
@@ -88,3 +89,20 @@ exports.createUserAdmin = userData => {
   userData.administrator = true;
   return upsert(userData);
 };
+
+exports.userAlbumsList = async req => {
+  try {
+    if (req.body.decode.administrator !== true) {
+      const user = await getUser(req.body.decode.email);
+      if (Number(user.id) !== Number(req.params.user_id)) {
+        throw error.userAlbumsListError('you can only see your albums');
+      }
+    }
+    const getAlbumsList = await servicesAlbums.getAlbumsListByIdUser(req.params.user_id);
+    return getAlbumsList;
+  } catch (err) {
+    throw error.userAlbumsListError(err);
```

Comment:
when you `throw error.userAlbumsListError(err);`, this line is overwriting the `throw` in line 98.